src/algebra/big_operators/finprod.lean

```diff
@@ -285,6 +285,15 @@ end
   ∏ᶠ i, f i = ∏ i in s, f i :=
 finprod_eq_prod_of_mul_support_subset _ $ λ x hx, h $ hf.mem_to_finset.2 hx
 
+@[to_additive] lemma finprod_eq_prod_of_mul_support_to_finset_subset'
```

Comment:
Should the `to_finset` be in the name? It doesn't explicitly feature in the statement.

bors d+